Users/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import User, Role
+
+admin.site.register(User)
```

Comment:
Admin add user form should be organized and should have password, password confirm fields and password reset option